tensorflow/lite/tools/optimize/quantization_utils.cc

```diff
@@ -85,6 +85,42 @@ void GetAsymmetricQuantizationParams(
   quantization_params->zero_point = std::vector<int64_t>(1, zero_point);
 }
 
+void GetSymmetricQuantizationParams(
+    float min, float max, const int half_quant_range,
+    QuantizationParametersT* quantization_params) {
+  // Adjust the boundaries to guarantee 0 is included.
+  min = std::min(min, 0.0f);
+  max = std::max(max, 0.0f);
+  const float scale = std::max(std::abs(max), std::abs(min)) / half_quant_range;
+  quantization_params->min = std::vector<float>(1, min);
```

Comment:
@wwwind can you please check below error 

`quantization_utils.cc:95:11: error: unused variable 'zero_point' [-Werror,-Wunused-variable]
  int64_t zero_point = 0;`

Reply:
Hi @rthadur It's has been fixed a little bit early in this commit
https://github.com/tensorflow/tensorflow/pull/36251/commits/01b30fa03f636396f977a76628123199b772463a
Could we please re-run what  has been failing ?
Thanks!